src/scaffoldmaker/meshtypes/meshtype_3d_bladder1.py

```diff
@@ -0,0 +1,770 @@
+                'Vessel inner diameter': 0.15,
+                'Vessel wall thickness': 0.04,
+                'Vessel angle 1 degrees': 0.0,
+                'Vessel angle 1 spread degrees': 0.0,
+                'Vessel angle 2 degrees': 0.0,
+                'Use linear through vessel wall': True,
+                # 'Use cross derivatives' : False,
+                'Refine': False,
+                'Refine number of elements around': 4,
+                'Refine number of elements along': 4,
+                'Refine number of elements through wall': 1
+                },
+            }),
+        'Ostium Rat 1': ScaffoldPackage(MeshType_3d_ostium1, {
+            'scaffoldSettings': {
+                'Number of vessels': 1,
+                'Number of elements across common': 2,
+                'Number of elements around ostium': 8,
+                'Number of elements along': 1,
+                'Number of elements through wall': 1,  # not implemented for > 1
+                'Unit scale': 1.0,
+                'Outlet': False,
+                'Ostium diameter': 0.15,
+                'Ostium length': 0.05,
+                'Ostium wall thickness': 0.02,
+                'Ostium inter-vessel distance': 0.8,
+                'Ostium inter-vessel height': 0.0,
+                'Use linear through ostium wall': False,
+                'Vessel end length factor': 1.0,
+                'Vessel inner diameter': 0.04,
+                'Vessel wall thickness': 0.01,
+                'Vessel angle 1 degrees': 0.0,
+                'Vessel angle 1 spread degrees': 0.0,
+                'Vessel angle 2 degrees': 0.0,
+                'Use linear through vessel wall': True,
+                # 'Use cross derivatives' : False,
+                'Refine': False,
+                'Refine number of elements around': 4,
+                'Refine number of elements along': 4,
+                'Refine number of elements through wall': 1
+                },
+            })
+        }
+
+    @staticmethod
+    def getName():
+        return '3D Bladder 1'
+
+    @staticmethod
+    def getParameterSetNames():
+        return [
+            'Default',
+            'Cat 1',
+            'Rat 1']
+
+    @classmethod
+    def getDefaultOptions(cls, parameterSetName='Default'):
+        if 'Rat' in parameterSetName:
+            ostiumOption = cls.ostiumDefaultScaffoldPackages['Ostium Rat 1']
+        else:
+            ostiumOption = cls.ostiumDefaultScaffoldPackages['Ostium Cat 1']
+
+        options = {
+            'Number of elements up 1': 8,
+            'Number of elements up 2': 16,
+            'Number of elements around': 8,  # should be even
+            'Number of elements through wall': 1,
+            'Height': 5.0,
+            'Major diameter': 3.0,
+            'Minor diameter': 3.0,
+            'Urethra radius': 0.5,
+            'Bladder wall thickness': 0.05,
+            'Number of elements around ostium': 8,# implemented for 8
```

Comment:
Scaffoldmaker style is to put the 'Number' options together at the top of the list, 'Refine' at the bottom.
List each section in alphabetic order (I make an exception only where list so big that related options need to be adjacent to each other).